mongodb.py

```diff
@@ -265,6 +265,17 @@ def do_server_status(self):
             for t in ['accesses', 'misses', 'hits', 'resets', 'missRatio']:
                 self.submit('counter', 'indexCounters.' + t, index_counters[t])
 
+        # WiredTiger
+        if 'wiredTiger' in server_status:
+            self.submit('gauge', 'wiredTiger.cache.bytesCurrentlyInTheCache',
+                        server_status['wiredTiger']['cache']['bytes currently in the cache'])
```

Comment:
I'm a bit wary of these index lookups since they will throw an exception if any of these keys don't exist.  Would be better to use `.get(..., {}).get(...)` or wrap this added block in an exception handler for IndexError.